src/software/ai/hl/stp/play/corner_kick_play_test.py

```diff
@@ -83,9 +84,25 @@ def test_corner_kick_play_bottom_left(simulated_test_runner):
             }
         ],
         inv_always_validation_sequence_set=[[]],
-        inv_eventually_validation_sequence_set=[[]],
+        inv_eventually_validation_sequence_set=[
+            [
+                BallAlwaysStaysInRegion(
```

Comment:
we should also check that the robot eventually kicks the ball

Reply:
Following up on this. It seems like this validation causes the test to fail in its current form. Trying to guarantee that the ball always stays in field is fairly difficult since it's easy to kick the ball slighly in the wrong direction. 

I suggest running the test and observing how it looks like and updating the validations accordingly. Specifically, in the second test the ball does not get kicked. We could add a TODO for that (I think there are already some issues for corner kick play being broken)